packages/reader-api/src/resultFile.ts

```diff
@@ -11,13 +11,16 @@ import {
   statSync,
 } from "node:fs";
 import "node:fs/promises";
-import { basename, extname } from "node:path";
+import { basename } from "node:path";
 import type { Readable } from "node:stream";
 import { pipeline } from "node:stream/promises";
 import { detectContentType } from "./detect.js";
+import { extension } from "./utils.js";
 
 export abstract class BaseResultFile implements ResultFile {
   fileName: string;
+  extension: string | false = false;
```

Comment:
Why we don't use `null` or `undefined` there?

Reply:
I guess `false` is more verbose